pkg/utils/keypath/absolute_key_path.go

```diff
@@ -61,10 +61,18 @@ const (
 	// "%08d" adds extra padding to make encoded ID ordered.
 	// Encoded ID can be decoded directly with strconv.ParseUint. Width of the
 	// padded keyspaceID is 8 (decimal representation of uint24max is 16777215).
-	gcSafePointPathFormat        = "/pd/%d/gc/safe_point/"                       // "/pd/{cluster_id}/gc/safe_point"
-	gcSafePointServicePathFormat = "/pd/%d/gc/safe_point/service/%s"             // "/pd/{cluster_id}/gc/safe_point/service/{service_id}"
-	gcSafePointV2PathFormat      = "/pd/%d/keyspaces/gc_safe_point/%08d"         // "/pd/{cluster_id}/keyspaces/gc_safe_point/{keyspace_id}"
-	serviceSafePointV2PathFormat = "/pd/%d/keyspaces/service_safe_point/%08d/%s" // "/pd/{cluster_id}/keyspaces/service_safe_point/{keyspace_id}/{service_id}"
+	gcStateRevisionPathFormat     = "/pd/%d/gc/gc_state_revision"         // "/pd/{cluster_id}/gc/gc_state_revision"
+	gcSafePointPathFormat         = "/pd/%d/gc/safe_point"                // "/pd/{cluster_id}/gc/safe_point"
```

Comment:
How about naming this variable globalGCSafePointPathFormat directly? Other developers might have questions about the relationship between gcSafePointPathFormat and keyspaceGCSafePointPathFormat: keyspaceGCSafePointPathFormat.